skops/hub_utils/_hf_hub.py

```diff
@@ -151,6 +151,11 @@ def _create_config(
         "text-regression",
     ],
     data,
+    save_format: Literal[  # type: ignore
```

Comment:
```suggestion
    model_format: Literal[  # type: ignore
```

I think `model_format` is more understandable that `save_format` in this case.